topics/monero/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: monero-cryptocurrency, monero-crypto
+created_by: Monero
+display_name: Monero
+github_url: https://github.com/monero-project
+released: April 18th, 2014
+short_description: Monero is a private, fungible, open source, decentralized cryptocurrency.
+topic: monero
+url: https://getmonero.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Monero_(cryptocurrency)
+---
```

Comment:
Can you add a full description after this line? Just something that expands on what the "short_description" says. Maybe compare Monero to other cryptocurrencies.

Reply:
Okay, I will. Thanks

Reply:
Done :)